src/layout/label.js

```diff
@@ -0,0 +1,46 @@
+import { range } from '../util/scale';
+import { rebindAll } from 'd3fc-rebind';
+import { label as labelLayout } from 'd3fc-label-layout';
+
+export default function(strategy) {
+
+    var adaptee = labelLayout(strategy);
+
+    var xScale = null,
+        yScale = null;
+
+
+    var label = function(selection) {
+        // automatically set the bounds of the strategy based on the scale range
+        if (strategy && strategy.bounds && xScale && yScale) {
```

Comment:
That's a good question, and why I defaulted to `null` above. I think the answer is 'no', it doesn't make sense to use it without scale - in that case you just use the non-adapted layout. I'll simplify this code a little ...